### PR TITLE
Group aa

### PR DESCRIPTION
**Pull request by Crownolar**
Pull Request Created

### CODE REVIEW

File: groupA.css [hunk 1]
```diff
@@ -7,5 +7,5 @@
 body{
     width: 100%;
     height: 100vh;
-    background: red;
+    background: aliceblue;
```
Comment (Yewande-Adewale) [inline groupA.css:10]: change to blue

**Review by Yewande-Adewale — CHANGES_REQUESTED** [review]

**Review by Yewande-Adewale — DISMISSED** [review]

File: groupA.css (hunk 1 above)
Comment (Crownolar) [reply groupA.css:10]: ok, on it

File: index.html [hunk 2]
```diff
@@ -6,5 +6,8 @@
     <link rel="stylesheet" href="./groupA.css">
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <h1>The Daughter universe Theory</h1>
+    <p>The Daughter universe theoory state that .................................</p>
```
Comment (Peterjoseph489) [inline index.html:11]: Change to  legend of the seeeker

**Review by Peterjoseph489 — CHANGES_REQUESTED** [review]

**Review by onyenze — APPROVED** [review]

**Review by onyenze — DISMISSED** [review]

**Review by Peterjoseph489 — APPROVED** [review]

**Review by Yewande-Adewale — COMMENTED** [review]
ok

**Review by Kennedy2042 — APPROVED** [review]

**Review by Yewande-Adewale — APPROVED** [review]

**Pull request merged**